### PR TITLE
Add CTAs to the Docs pages

### PR DESCRIPTION
**Pull request by usha-mandya**
Signed-off-by: Usha Mandya <usha.mandya@docker.com>

Add CTAs to the popular pages on the Docs site

### CODE REVIEW

**Comment by thaJeztah** [comment]
should we have includes for these (if we're gonna be using the same CTA on various pages?)

**Comment by thaJeztah** [comment]
Also wondering if these are things that could be / should be inserted through GTM 🤔

**Comment by usha-mandya** [comment]
> should we have includes for these (if we're gonna be using the same CTA on various pages?

@thaJeztah Includes are a good idea. Wondering how we can handle relative linking to internal docs without adding the full URL? For example, if we have to link to the Subscription docs from the Desktop overview page and also from the Windows install page

**Review by stephanierifai — APPROVED** [review]
LGTM!

File: desktop/windows/install.md [hunk 1]
```diff
@@ -12,6 +12,11 @@ redirect_from:
 - /install/windows/ee-preview/
 ---
 
+> **Update to the Docker Desktop terms**
+>
+> Professional use of Docker Desktop in large organizations (more than 250 employees or more than $10 million in revenue) requires users to have a paid Docker subscription. While the effective date of these terms is August 31, 2021, there is a grace period until January 31, 2022 for those that require a paid subscription. For more information, see [Docker Desktop License Agreement](../../subscription/index.md#docker-desktop-license-agreement).
```
Comment (jerae-duffin) [inline desktop/windows/install.md:17]: Minor: comma after January 31, 2022,

File: get-docker.md [hunk 2]
```diff
@@ -8,6 +8,10 @@ redirect_from:
 - /installation/
 ---
 
+> **Update to the Docker Desktop terms**
+>
+> Professional use of Docker Desktop in large organizations (more than 250 employees or more than $10 million in revenue) requires users to have a paid Docker subscription. While the effective date of these terms is August 31, 2021, there is a grace period until January 31, 2022 for those that require a paid subscription. For more information, see the blog [Docker is Updating and Extending Our Product Subscriptions](https://www.docker.com/blog/updating-product-subscriptions/){: target="_blank" rel="noopener" class="_" id="dkr_docs_subscription_btl"}.
+{: .important}
```
Comment (jerae-duffin) [inline get-docker.md:14]: Minor change: comma after January 31, 2022,

**Review by jerae-duffin — APPROVED** [review]
LGTM!

**Pull request merged**